rosa_planning/rosa-understanding-terraform.adoc

```diff
@@ -0,0 +1,34 @@
+:_content-type: ASSEMBLY
+include::_attributes/attributes-openshift-dedicated.adoc[]
+
+[id="rosa-understanding-terraform"]
+= Using Terraform to install ROSA clusters
+:context: rosa-understanding-terraform
+
+toc::[]
+
+Terraform is an infrastructure-as-code tool that provides a way to configure your resources once and replicate those resources as desired. Terraform accomplishes these creation tasks by using a declarative language. You declare what you want the final state of the infrastructure resource, and Terraform creates these resources to your specifications.
+
+include::modules/sd-terraform-prereqs.adoc[leveloffset=+1]
+
+[discrete]
+[role="_additional-resources"]
+[id="additional-resources_rosa-terraform-prereq"]
+=== Additional resources
+
+* See xref:../rosa_architecture/rosa-sts-about-iam-resources.adoc[About IAM resources for ROSA clusters that use STS] for information about the AWS account role.
```

Comment:
Earlier on this page it was written as "AWS account roles" plural. Should "role" be plural or singular?